server/src/main/resources/resourceBundles/messages_en.properties

```diff
@@ -11,6 +11,11 @@ CobolVisitor.misspelledWord=A misspelled word, maybe you want to put %s
 CobolVisitor.paragraphNotDefined=The following paragraph is not defined: %s
 CobolVisitor.progIDIssueMsg=There is an issue with PROGRAM-ID paragraph
 CobolVisitor.subroutineNotFound=%s: Subroutine not found
+CobolVisitor.theSameGroup=Redefined element should belong to the same group: %s
+CobolVisitor.redefinedContainValue=Redefined clause can not contain VALUE: %s
+CobolVisitor.redefineRedefined=Redefined variable cannot be redefined: %s
+CobolVisitor.redefineImmediatelyFollow=Redefined variable should immediately follow: %s
+CobolVisitor.levelsMustMatch=Redefined variable level must be the same: %s
```

Comment:
"Redefined" in this context means the variable to the right, e.g. `A` in `05 B REDEFINES A.`, so the error messages should be formulated accordingly.
@zeibura please, review it.

Reply:
Are we allowed to use the terms "redefining item" and "redefined item" like the IBM doc does? If so it seems to be:

1. The redefining and redefined items must belong to the same group
2. The redefining item cannot contain a VALUE clause
3. the IBM doc says the opposite. B REDEFINES A = A can contain a REDEFINES clause itself = saying B can't contain a REDEFINES clause is self-evidently false. Sure you've got this right?
4. actually not sure what this means, can you give me example code which would trigger this error?
5. The redefining and redefined items must have the same level

Reply:
@zeibura you are right, redefined item can be redefined. @temanbrcom I guess this check must be removed.

4. "One or more redefinitions of the same storage area are permitted. The entries that give the new descriptions of the storage area must immediately follow the description of the redefined area without intervening entries that define new character positions. Multiple redefinitions can, but need not, all use the data-name of the original entry that defined this storage area. "
````COBOL
             IDENTIFICATION DIVISION
             PROGRAM-ID. PERFTIMI.
             ENVIRONMENT DIVISION.
             DATA DIVISION.
             WORKING-STORAGE SECTION.
             01 WS-DATA-A PIC X(20) .
             01 WS-DATA-RED REDEFINES WS-DATA-A PIC X(20).
             01 WS-DATA-RED2 REDEFINES WS-DATA-A PIC X(20).
             01 WS-DATA-B} PIC X(20) .
             01 WS-DATA-RED3 REDEFINES WS-DATA-A PIC X(20). <--- ERROR
             PROCEDURE DIVISION.\n";

````

Reply:
Ah okay I see! So I think the error should be
REDEFINES line must immediately follow redefined item